pxr/imaging/garch/CMakeLists.txt

```diff
@@ -11,7 +11,6 @@ if(APPLE)
     set(GARCH_GLPLATFORMCONTEXT       glPlatformContextDarwin)
     set(GARCH_GLPLATFORMDEBUGWINDOW   glPlatformDebugWindowDarwin)
     set(GARCH_SOURCE_EXTENSION        mm)
-    FIND_LIBRARY(APPKIT_LIBRARY AppKit)
```

Comment:
I think it might make sense to add `set(GARCH_PLATFORM_LIBRARIES "-framework AppKit") here and ...

Reply:
Ah yeah that's a good idea. I'll make the change and update tomorrow.